great_expectations/datasource/datasource_serializer.py

```diff
@@ -9,12 +9,12 @@
 serializer = YAMLReadyDictDatasourceConfigSerializer()
 serialized_value = serializer.serialize(datasource_config)
 """
+from great_expectations.core.configuration import AbstractConfig
```

Comment:
If this is just for type checking I would go the `if TYPE_CHECKING:` route.

```python
from typing import TYPE_CHECKING:

if TYPE_CHECKING:
   from great_expectations.core.configuration import AbstractConfig


...

class YAMLReadyDictDatasourceConfigSerializer(AbstractConfigSerializer):
    def serialize(self, obj: 'AbstractConfig') -> dict:

```

Reply:
Good call! Done 🙇